ddtrace/propagation/_database_monitoring.py

```diff
@@ -25,23 +26,27 @@
 log = get_logger(__name__)
 
 
-def _default_sql_injector(dbm_comment, sql_statement):
-    # type: (str, str) -> str
+def default_sql_injector(dbm_comment, sql_statement):
+    # type: (Union[str, bytes], Union[str, bytes]) -> Union[str, bytes]
     try:
+        if isinstance(sql_statement, bytes):
+            try:
+                dbm_comment = dbm_comment.encode("utf-8", errors="strict")
+            except UnicodeDecodeError:
+                dbm_comment = b""
         return dbm_comment + sql_statement
-    except TypeError:
+    except Exception:
```

Comment:
what else can get thrown here? wouldn't it always be a `TypeError`?

if we want to keep this as `Exception`, then we should probably include the Exception message in the error log, otherwise we cannot know why this try/except failed.

Reply:
Let's switch it back to `TypeError` now that Emmet's change catches the `UnicodeDecodeError`.

Reply:
> or could just do except (TypeError, ValueError):

I meant, we could do this instead of the inner try/except block.